docs/OV_Runtime_UG/preprocessing_overview.md

```diff
@@ -131,17 +131,18 @@ Now we can define sequence of preprocessing steps:
 
 @endsphinxtabset
 
-Here:
- - Convert U8 to FP32 precision
- - Convert current color format (BGR) to RGB
- - Resize to model's height/width. **Note** that if model accepts dynamic size, e.g. {?, 3, ?, ?}, `resize` will not know how to resize the picture, so in this case you should specify target height/width on this step. See also <code>ov::preprocess::PreProcessSteps::resize()</code>
- - Subtract mean from each channel. On this step, color format is RGB already, so `100.5` will be subtracted from each Red component, and `101.5` will be subtracted from `Blue` one.
- - Divide each pixel data to appropriate scale value. In this example each `Red` component will be divided by 50, `Green` by 51, `Blue` by 52 respectively
- - **Note:** last `convert_layout` step is commented out as it is not necessary to specify last layout conversion. PrePostProcessor will do such conversion automatically
+Perform as follows:
 
-### Integrate steps into model
+   1. Convert `U8` to `FP32` precision.
+   2. Convert current color format (`BGR`) to `RGB`.
+   3. Resize to `height`/`width` of a model. Be aware that if a model accepts dynamic size, e.g. `{?, 3, ?, ?}`, `resize` will not know how to resize the picture. Therefore, in this case, target `height`/`width` should be specified. See also `ov::preprocess::PreProcessSteps::resize()`.
+   4. Subtract mean from each channel. In this step, color format is RGB already, so `100.5` will be subtracted from each `Red` component, and `101.5` will be subtracted from `Blue` one.
+   5. Divide each pixel data to appropriate scale value. In this example, each `Red` component will be divided by 50, `Green` by 51, `Blue` by 52 respectively.
+   6. Note that the last `convert_layout` step is commented out as it is not necessary to specify the last layout conversion. PrePostProcessor will do such conversion automatically.
 
-We've finished with preprocessing steps declaration, now it is time to build it. For debugging purposes it is possible to print `PrePostProcessor` configuration on screen:
+### Integrating Steps into a Model
+
+Build it when the preprocessing has been finished. It is possible to print `PrePostProcessor` configuration on screen for debugging purposes:
```

Comment:
```suggestion
Model can be finally build, once the preprocessing steps has been finished. It is possible to print `PrePostProcessor` configuration on screen for debugging purposes:
```

Reply:
> 

```suggestion
Model can be finally built, once the preprocessing steps have been finished. It is possible to print `PrePostProcessor` configuration on screen for debugging purposes:
```

We use past participle form of a verb in the passive voice.